plugins/org.wso2.developerstudio.eclipse.gmf.esb/src/org/wso2/developerstudio/eclipse/gmf/esb/EsbPackage.java

```diff
@@ -42000,6 +42029,17 @@
 	EReference getLoadBalanceEndPoint_MediatorFlow();
 
 	/**
+	 * Returns the meta object for the attribute '{@link org.wso2.developerstudio.eclipse.gmf.esb.LoadBalanceEndPoint#isBuildMessage <em>Build Message</em>}'.
```

Comment:
These comments exceeds 120 indentation limit. Can we format this?

Reply:
Generated codes from GMF. These code segments will be replaced in the same way when we generate it again even if we format them now.